app/controllers/api/subcollections/orchestration_stacks.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module OrchestrationStacks
+      def orchestration_stacks_query_resource(object)
+        object.respond_to?(:orchestration_stacks) ? object.orchestration_stacks : []
```

Comment:
Are there services that don't have orchestration stacks? If so, can you add a test to verify what happens when it doesn't?